pkg/migrations/migrations.go

```diff
@@ -747,33 +747,37 @@ func Migrate() {
 							if err != nil {
 								return err
 							}
-							continue
+							// common.Log.Infof("Updated scene %s", scene.SceneID)
 						}
 					}
 				}
 
+				var scenes_tngf []models.Scene
+				err = tx.Where("site = ?", "Tonight's Girlfriend VR").Find(&scenes_tngf).Error
+				if err != nil {
+					return err
+				}
 				items := gjson.Get(tngf_json, "@this")
-				for _, scene := range scenes {
+				for _, scene := range scenes_tngf {
 					for _, scenejson := range items.Array() {
 						if scene.ReleaseDateText == scenejson.Get("date").String() {
 							scene.ReleaseDateText = scenejson.Get("date").String()
 							scene.ReleaseDate, _ = time.Parse(time.RFC3339, scene.ReleaseDateText+"T00:00:00-04:00")
-							scene.SceneID = scenejson.Get("sceneid").String()
+							sceneID := scenejson.Get("sceneid").String()
 							scene.SceneURL = scenejson.Get("sceneurl").String()
 
 							// update all actions referring to this scene by its scene_id
-							err = tx.Model(&models.Action{}).Where("scene_id = ?", scene.SceneID).Update("scene_id", scene.SceneID).Update("release_date", scene.ReleaseDate).Update("release_date_text", scene.ReleaseDateText).Update("scene_url", scene.SceneURL).Error
+							err = tx.Model(&models.Action{}).Where("scene_id = ?", scene.SceneID).Update("scene_id", sceneID).Error
 							if err != nil {
 								return err
 							}
-
+							scene.SceneID = sceneID
```

Comment:
This will lead to duplicate scenes even sharing the same scene id if you've run the new scraper, right?